app/models/ckey2bibframe.rb

```diff
@@ -12,7 +12,8 @@ def convert
     catalogdump = "echo #{ckey} | #{Settings.symphony_catalogdump}"
 
     jar = 'lib/xform-marc21-to-xml-jar-with-dependencies.jar'
-    java_command = "java -Djava.security.egd=file:/dev/../dev/urandom -cp #{jar} edu.stanford.MarcToXMLStream"
+    java_command = "java -Djava.security.egd=file:/dev/../dev/urandom -cp #{jar} \
+                    edu.stanford.MarcToXMLStream -p config/settings/server.conf"
```

Comment:
Maybe all those strings could be wrapped in a method called `java_command` so that method encapsulates all the details for all time?